tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/contact/ContactRabbitMqMessage.java

```diff
@@ -0,0 +1,19 @@
+package com.linagora.tmail.contact;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public record ContactRabbitMqMessage(String bookId, String bookName, String contactId,
```

Comment:
Remember me to verify on friday that indeed message format is always the same...